src/resources/postcss/tickets-refresh.pcss

```diff
@@ -1,3 +1,24 @@
+@define-mixin tribe-ticket-tooltip {
+	position: relative;
+
+	.tooltip {
+		border-radius: 3px;
+		border: 1px solid #a3a3a3;
```

Comment:
the nitpickiest of nitpicks but ... move `border-radius` under `border` to main alphabetization